code/modules/mob/living/carbon/xenomorph/castes/behemoth/abilities_behemoth.dm

```diff
@@ -759,37 +747,32 @@
 			pixel_x += 18
 			pixel_y -= 8
 
-/datum/action/ability/activable/xeno/seismic_fracture
+/datum/action/ability/xeno_action/seismic_fracture
```

Comment:
why's this no longer an activatable?

Reply:
Because the ability was changed to no longer be targetable. It's now an AoE around the Behemoth, so it might as well just not be an activable anymore.